src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2492,7 +2492,7 @@ public abstract partial class Enum : System.ValueType, System.IComparable, Syste
         public string ToString([System.Diagnostics.CodeAnalysis.StringSyntaxAttribute("EnumFormat")] string? format) { throw null; }
         [System.ObsoleteAttribute("The provider argument is not used. Use ToString(String) instead.")]
         public string ToString([System.Diagnostics.CodeAnalysis.StringSyntaxAttribute("EnumFormat")] string? format, System.IFormatProvider? provider) { throw null; }
-        public static bool TryFormat<TEnum>(TEnum value, System.Span<char> destination, out int charsWritten, [System.Diagnostics.CodeAnalysis.StringSyntaxAttribute("EnumFormat")] System.ReadOnlySpan<char> format = default(System.ReadOnlySpan<char>)) where TEnum : struct { throw null; }
+        public static bool TryFormat<TEnum>(TEnum value, System.Span<char> destination, out int charsWritten, [System.Diagnostics.CodeAnalysis.StringSyntaxAttribute("EnumFormat")] System.ReadOnlySpan<char> format = default(System.ReadOnlySpan<char>)) where TEnum : struct, System.Enum { throw null; }
```

Comment:
I think this should be removed in the implementation instead. Notice that TryParse APIs on this type do not have the Enum contract.

Reply:
@stephentoub  what do you think?  You added the API with the constraint in https://github.com/dotnet/runtime/commit/62f3eb2878c3e82631b91dd02ee4fb655d23a2b1

Reply:
As with above this already fails at runtime already with:
```
System.Security.VerificationException: Method System.Enum.TryFormat: type argument 'System.DateTime' violates the constraint of type parameter 'TEnum'.
```

Reply:
We approved it with the constrain but I believe this was because it was proposed like that and it made sense to us. Not having the constraint seems fine.

Reply:
Ok, I'll remove the constraints from both ref and source and ensure that this API checks the T and throws an argument exception then (unless @stephentoub mentions otherwise).